app/app.go

```diff
@@ -128,6 +128,12 @@ import (
 	erc20client "github.com/evmos/evmos/v10/x/erc20/client"
 	erc20keeper "github.com/evmos/evmos/v10/x/erc20/keeper"
 	erc20types "github.com/evmos/evmos/v10/x/erc20/types"
+	ibcevmtx "github.com/evmos/evmos/v10/x/ibc/evm"
```

Comment:
nit, `tx` is not port of the path and it's nice to stay consistent with the path

```suggestion
	ibcevm "github.com/evmos/evmos/v10/x/ibc/evm"
```